.github/workflows/semantic-release.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: Semantic Release
+
+run-name: Semantic Release from ${{ github.ref_name }}
+
+on:
+  push:
+    branches:
+      # - "**"
+      - "TODO: Enable once version 1.0.0 is released."
```

Comment:
can we use the semantic release system from [makenew-tsmodule](https://github.com/seamapi/makenew-tsmodule)? I'm a bit confused by some of the things here e.g. the TODO

Reply:
https://github.com/seamapi/seam-cli/pull/14/commits/f87563308d2e7834e67bedb4e140e2ddc2333621